sqlglot/optimizer/helper.py

```diff
@@ -0,0 +1,8 @@
+from sqlglot import expressions as exp
+
+
+def quote(expression, arg):
```

Comment:
can we just have this as a property of expression.?

expression.identify

Reply:
Sure